lib/menu.rb

```diff
@@ -0,0 +1,30 @@
+class Menu
+  attr_accessor :menu
+
+  def initialize
+    @menu = {
+      "Spring roll" => 0.99,
+      "Char sui bun" => 3.99,
+      "Pork dumpling" => 2.99,
+      "Peking duck" => 7.99,
+    }
+  end
+
+  def show_menu
+    puts "---------------------------------------"
+    puts "*               MENU                  *"
+    puts "---------------------------------------"
+    @menu.each do |item, price|
+      puts "#{item} ........ #{price}".center(40)
+    end
+    puts "---------------------------------------"
+  end
+
+  def add2_menu
```

Comment:
personally i would add this to the takeaway class as i feel the takeaway is the one that adds new dishes to the menu